conf/core/ConfigOptions.asciidoc

```diff
@@ -144,6 +144,34 @@ See also:
 
  * _Estimate Pairwise Relationships_, <<hootalgo>>
 
+=== changeset.buffer
+
+* Data Type: double
+* Default Value: `0.0`
+
+Value in degrees by which the specified AOI (convert.bounding.box) will be expanded when calculating
+changesets.  Setting this too large with some datasets may greatly increase the changeset
+derivation time.
+
+=== changeset.allow.deleting.from.features
+
+* Data Type: bool
+* Default Value: `true`
+
+If true, changesets derived can issue delete statements for the 'from' dataset (first dataset
+passed to the changeset deriver).  If false, no delete statements will be issued for the 'from'
+dataset.
```

Comment:
I'm open to suggestions on this...I have no ideas regarding removing this setting and making the changesets work.  If it gets removed, then you're going to drop chunks of features in certain situations.

My idea is to leave it enabled, generate the changeset, then have iD use its change history to determine if the user manually deleted a reference features.  If so, iD would add the delete statement for the feature to the changeset before sending it.